*** ./smali/com/android/internal/statusbar/IStatusBar.smali	2013-04-30 04:51:39.747191000 -0500
--- /home/dhacker29/patchrom/xt926/temp/new_smali/framework.jar.out/./smali/com/android/internal/statusbar/IStatusBar.smali	2013-04-30 04:54:06.847191000 -0500
***************
*** 103,108 ****
--- 103,116 ----
      .end annotation
  .end method
  
+ .method public abstract setStatus(ILjava/lang/String;Landroid/os/Bundle;)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setSystemUiVisibility(II)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
